app/com/linkedin/drelephant/spark/fetchers/statusapiv1/StageStatus.java

```diff
@@ -0,0 +1,17 @@
+package com.linkedin.drelephant.spark.fetchers.statusapiv1;
```

Comment:
Adding a Java class in a Scala package looks weird.  Either create a Scala class here or create a separate package for this class.

Reply:
In apache spark documentation, StageStatus is a .java class hence, to keep things uniform, I will create a new package for this class.